src/renderer/fonts/caching_shaper.rs

```diff
@@ -373,16 +351,10 @@ impl CachingShaper {
         grouped_results
     }
 
-    pub fn adjust_font_cache_size(&self) {
-        let current_font_cache_size = font_cache_limit() as f32;
-        let percent_font_cache_used = font_cache_used() as f32 / current_font_cache_size;
-        if percent_font_cache_used > 0.9 {
-            warn!(
-                "Font cache is {}% full, increasing cache size",
-                percent_font_cache_used * 100.0
-            );
-            set_font_cache_limit((percent_font_cache_used * 1.5) as usize);
-        }
+    pub fn cleanup_font_cache(&self) {
```

Comment:
This is the only change that gives me pause. How can we be sure that the cache size constant is big enough?

Reply:
There are a few reasons:

1. Even if the cache is too small it will still work, it just becomes a lot slower. 
2. The default cache size was 2 MB before, and we never hit that. We would have known, since there was a bug in the growing code which sets it to 1 or 2 bytes, it multiplied by percent not bytes.
3. The temporal cache during the scrolling is much bigger than the statically needed cache for just displaying the text, since it needs to store all subpixel positions encountered during the scrolling. With the tracy logger the total cache size never got bigger than around 2MB when scrolling completely through a few big buffers, so with the current code. The cleanup condition was not even met when storing all that in the cache. It would perhaps be worth doing the same with some Asian languages and a big font, and a completely full 4k+ screen, but I'm pretty sure that it's still way more than enough.
4. Even if we clean up all the cache needed for scrolling it would just be as slow as when you first start scrolling, which still has a pretty acceptable performance IMO.